microsoft-edge/devtools-guide-chromium/ie-mode/index.md

```diff
@@ -1,24 +1,24 @@
 ---
 description: IE mode and the Microsoft Edge (Chromium) DevTools
-title: Internet Explorer mode and the DevTools
+title: Use the DevTools in Internet Explorer Mode
```

Comment:
```suggestion
title: Use the DevTools in Internet Explorer mode
```
Lowercase 'm' like at "What is Internet Explorer (IE) mode?" https://docs.microsoft.com/en-us/deployedge/edge-ie-mode